storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,277 @@
+                Voor Weesp kijkt u op{' '}
+                <Link color="inverse" href="#" variant="inline">
+                  Parkeertarieven stadsgebied Weesp
+                </Link>
+                .
+              </Paragraph>
+              <UnorderedList className="ams-mb--lg" color="inverse">
+                <UnorderedList.Item>Vul een adres in om de mogelijkheden te bekijken</UnorderedList.Item>
+                <UnorderedList.Item>
+                  Als u op een feestdag wilt parkeren, bekijk dan ook{' '}
+                  <Link color="inverse" href="#" variant="inline">
+                    Parkeren op feestdagen
+                  </Link>
+                  .
+                </UnorderedList.Item>
+              </UnorderedList>
+              <SearchField>
+                <SearchField.Input />
+                <SearchField.Button />
+              </SearchField>
+            </Grid.Cell>
+          </Grid>
+        </Spotlight>
+        <Image
+          alt=""
+          aspectRatio="x-wide"
+          className="ams-mb--xl"
+          key={1}
+          src="http://www.parkerenindestad.nl/images/amsterdam/parkeren-in-amsterdam.jpg"
+          style={{ inlineSize: '100%' }}
+        />
+      </>
+    ),
+  },
+}
+
+export const SubnavigationPage: StoryObj = {
+  args: {
+    children: (
+      <>
+        <Grid paddingBottom="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+            <PageHeading className="ams-mb--md">Overzichtspagina content / Subnavigatiepagina</PageHeading>
+            <Paragraph size="large">
+              Amsterdam wil een nieuwe traditie starten om met oud en nieuw naar een centrale nieuwjaarsviering te gaan
+              in plaats van zelf vuurwerk af te steken.
+            </Paragraph>
+          </Grid.Cell>
+        </Grid>
+        <Image alt="" src="https://picsum.photos/1600/900" />
+        <Grid paddingVertical="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <Heading className="ams-mb--sm" level={2}>
+              L2 Paragraaf titel
+            </Heading>
+            <Paragraph>{exampleParagraph()}</Paragraph>
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <NavigationColumn sections={burgerzakenLinks[0].slice(0, 2)} />
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }}>
+            <NavigationColumn sections={burgerzakenLinks[1].slice(0, 2)} />
+          </Grid.Cell>
+          <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+            <Card>
+              <Heading className="ams-mb--sm" level={2} size="level-4">
+                {exampleHeading()}
+              </Heading>
+              <Paragraph>{exampleParagraph()}</Paragraph>
+              <Link href="#" variant="standalone">
```

Comment:
Because they are in a flex context, these links get the width of their container, even though their text is smaller. This leads to clickable white space, which is a little confusing.